be/src/exec/tablet_sink.cpp

```diff
@@ -689,7 +693,10 @@ void OlapTableSink::_convert_batch(RuntimeState* state, RowBatch* input_batch, R
                     _number_filtered_rows++;
                     break;
                 }
+                dst_tuple->set_null(slot_desc->null_indicator_offset());
+                continue;
             }
+            dst_tuple->set_not_null(slot_desc->null_indicator_offset());
```

Comment:
If this slot is not nullable, `set_not_null` may cause problem

Reply:
done